src/platforms/ios/ioscontroller.swift

```diff
@@ -200,6 +200,7 @@ public class IOSControllerImpl : NSObject {
                 //Used to create a VPN configuration without connecting during onboarding
                 if isOnboarding {
                     IOSControllerImpl.logger.info(message: "Finishing onboarding... do not turn on the VPN after gaining permission")
+                    failureCallback()
```

Comment:
Since this is now being intentionally used as a `disconnectCallback`, what do you think about renaming it? And updating the log line in the callback?

Reply:
Agree - updated.